### ISSUE TITLE
Tab indents are sometimes misaligned after reopening page later, breaking tab-based table column alignment

### ISSUE THREAD

**Issue by hsandt** [zim-desktop-wiki/zim-desktop-wiki#2419]
**Describe the bug**
I thought I had already reported this but I cannot find any issue related to this, so I'm opening a ticket anyway.

I write a lot of pseudo-tables using tab indents to align column content.
After a while though, if I reopen the page, I notice that many entries are suddenly misaligned. As if the tab size changes. This may be caused by resolution change.
I noticed it happens when switching OS (Windows and Linux), but sometimes even on same OS. Maybe switching monitor (1440p to 4K), but couldn't find anything consistent.

It seems to occur primarily when the first column alternates between long and short entries (like hyphen `-`). Maybe we're at the limit of pushing the next column text back and some sensitive parameter changes so the column is unstable and moves after certain Zim updates? But it also occurs even without updating Zim...

**To Reproduce**
1. Create a pseudo table using tab indents to align "cells". Use "cell" text of various sizes
2. Do something else, change OS, monitor, upgrade Zim, etc.
3. Come back to the page
4. See how column alignment is broken

**Expected behavior**
Tab indent alignment should be stable.

**Screenshots**

When making the pseudo-table with indents:
![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/1067772/760c49fc-1f07-4b42-b689-640c5be31498)

After a while, when coming back to page:
![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/1067772/b974efa6-0a87-47c3-8da6-140ed1ed0f44)

**Environment (please complete the following information):**
- Version 0.75.2
- OS: Linux Ubuntu 22.04 with Unity desktop
- Language: English

**Additional context**
I change monitors often, OS less often, upgrade Zim sometimes... Issue is very visible when switching between Linux and Windows, but even after sticking to one OS, it occurred. And I assume it would stabilize after a few version upgrades anyway.

**Comment by introt** [comment]
> I write a lot of pseudo-tables using tab indents to align column content.

Sorry, as Jaap stated in your original issue:

> Yes, afraid that is "won't fix" - never expect tab alignment to work without fixed wi[d]th font

Tabs are too fragile and not portable due to different fonts inside apps as well as renderers between apps.

**Comment by hsandt** [comment]
OK, I see. I tried Format > Verbatim to use a mono font, unfortunately characters were far too wide for my purpose (my description text becomes too long and goes on to next line too quickly). Then I tried to change the scale on both code and pre following https://lists.launchpad.net/zim-wiki/msg01929.html. Multi-line `pre` verbatim still had an extra initial indent that made me waste space, so I also reduced indent from 20 to 8 (it's not a number of characters, but a pure measurement in some spatial unit):

```
[Tag code]
...
scale=0.8

[Tag pre]
...
indent=8
scale=0.8
```

and it worked! I also changed the font family to something I liked better.

Note that https://zim-wiki.org/manual/Help/Config_Files.html says scale is an integer, but this is incorrect: even default config uses fractional values for some scales.

This should help with my other indent-based pseudo-table issues like https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/2418.
